papers/oliver_beckstein/oliver_beckstein.rst

```diff
@@ -93,6 +93,32 @@ The user interface and modular design work equally well in complex scripted work
 MDAnalysis is available in source form under the GNU General Public License v2 from GitHub https://github.com/MDAnalysis/mdanalysis and PyPi;  conda packages are also available. The documentation is extensive http://docs.mdanalysis.org including an introductory tutorial http://www.mdanalysis.org/MDAnalysisTutorial/ and a very friendly and welcoming user and developer community. 
 
 
+New data Structures
+-------------------
```

Comment:
I feel that background for Universe/AtomGroup is missing. It should be added in another chapter explaining the basics.